README.md

```diff
@@ -4,17 +4,17 @@
 </p>
 
 <h1 align="center">Meteor</h1>
-<p align="center">A Minecraft Fabric Utility Mod for anarchy servers.</p>
+<p align="center">A Minecraft Fabric Utility Mod for Anarchy Servers </p>
```

Comment:
it'd be better to give this proper capitalization instead of starting each word with a capital letter